<!-- 
  Launch file for global mapping
  Parameters:
  - Use debug_mode:=true to publish debug information
-->

<launch>
  <arg name="debug_mode" default="false" />

  <node pkg="height_mapping" type="global_mapping_node" name="height_mapping_global" output="screen">
    <rosparam command="load" subst_value="true" file="$(find height_mapping)/config/global_mapping_node.yaml" />
    <param name="node/debug_mode" value="$(arg debug_mode)" />
  </node>

</launch>